gradle/versions.gradle

```diff
@@ -43,7 +43,7 @@ ext {
     uiAutomatorVersion = "2.2.0"
     msal4jVersion = "1.10.0"
     mseberaApacheHttpClientVersion = "4.5.8"
-    yubikitAndroidVersion = "2.0.0"
+    yubikitAndroidVersion = "2.1.0-alpha.1"
```

Comment:
Is the alpha version of their library considered stable?

Reply:
no, but I don't think any major changes will be added before their next stable release. This CBA feature will use the alpha release for private preview, and then the final stable release will replace it before GA (when it'll be merged into dev).